theano/gpuarray/opt.py

```diff
@@ -264,8 +267,18 @@ def add_requirements(self, fgraph):
         fgraph.attach_feature(toolbox.ReplaceValidate())
 
     def apply(self, fgraph):
+        change_tracker = ChangeTracker()
         mapping = {}
-
+        global_process_count = {}
```

Comment:
process_count is used to count the local process. You don't need to duplicate that.